vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -66,11 +66,21 @@ Dialogs.disable=Don't show this again
 Dialogs.show_details=Show details
 Dialogs.hide_details=Hide details
 Dialogs.dont_show_again=Don't show this again
+
+Dialogs.removed_code_title=Defunct Custom Code
+Dialogs.removed_code_heading=Defunct Custom Code
+Dialogs.removed_code_message=This module uses VASSAL classes, methods, or fields which were deprecated more than a year ago and have since been removed. In order to run this module, uses of the classes, methods, or fields listed below must be replaced and the custom code recompiled.
+
+Dialogs.deprecated_code_title=Obsolete Custom Code
+Dialogs.deprecated_code_heading=Obsolete Custom Code
+Dialogs.deprecated_code_message=This module uses VASSAL classes, methods, or fields which are deprecated and will be removed in some future version. The uses of the classes, methods, or fields listed below must be replaced and the custom code recompiled before these classes, methods, or fields are removed.
```

Comment:
This module contains "custom code" that uses VASSAL classes, methods, or fields which are deprecated and will be removed in some future version. 

IF YOU ARE A MODULE USER - you can run the module for now, but you may want to let the module designer/maintainer know that their custom code needs updating.

IF YOU ARE THE MODULE DESIGNER/MAINTAINER - The uses of the classes, methods, or fields listed below should be replaced and the custom code recompiled before these classes, methods, or fields are removed. If you are not the original maintainer and do not have the knowledge of how to do this, please post a message in the Technical Support forums at https://vassalengine.org/forum